zvariant/src/value.rs

```diff
@@ -738,3 +891,165 @@ impl<'a> Type for Value<'a> {
         Signature::from_static_str_unchecked(VARIANT_SIGNATURE_STR)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::HashMap;
+
+    use super::*;
+
+    #[test]
+    fn value_display() {
+        assert_eq!(
+            Value::new((
+                255_u8,
+                true,
+                -1_i16,
+                65535_u16,
+                -1,
+                1_u32,
+                -9223372036854775808_i64,
+                18446744073709551615_u64,
+                (-1., 1.0, 11000000000., 1.1e-10)
+            ))
+            .to_string(),
+            "(byte 0xff, true, int16 -1, uint16 65535, -1, uint32 1, int64 -9223372036854775808, uint64 18446744073709551615, (-1., 1., 11000000000., 0.00000000011))"
```

Comment:
Strings literals can be broken over multiple lines using `\` suffix.

Reply:
Thanks! Did not knew of that